### PR TITLE
Add Ansible, Puppet, and Chef to Alloy install documentation

### PR DESCRIPTION
**Pull request by clayton-cornell**
#### PR Description

This PR adds Ansible, Chef, and Puppet to the Alloy install documentation.  These install topics were removed in the 1.0 release.

#### PR Checklist

- [ ] CHANGELOG.md updated
- [ ] Documentation added
- [ ] Tests updated
- [ ] Config converters updated

### CODE REVIEW

File: docs/sources/set-up/install/puppet.md [hunk 1]
```diff
@@ -27,7 +24,7 @@ You can use Puppet to install and manage {{< param "PRODUCT_NAME" >}}.
 
 To add {{< param "PRODUCT_NAME" >}} to a host:
 
-1. Ensure that the following module dependencies are declared and installed:
+1. Make sure that the following module dependencies are declared and installed:
 
     ```json
     {
```
Comment (wildum) [inline docs/sources/set-up/install/puppet.md:30]: below is a ref to grafana_agent: `class grafana_agent::grafana_agent_flow ()`

Comment (clayton-cornell) [reply docs/sources/set-up/install/puppet.md:30]: I missed it :-(  Fixed in a new commit.

**Comment by wildum** [comment]
should we make sure that the recommended steps are working with Alloy?

**Comment by clayton-cornell** [comment]
> should we make sure that the recommended steps are working with Alloy?

Yes definitely. I don't have a VM set up to test deployments on my side though.

The Ansible collection seems up to date - Alloy was added a while back and several updates.

**Comment by clayton-cornell** [comment]
@wildum do you think we are "safe" enough to re-publish this info as-is? It was validated on the last version of Agent Flow.  I don't have an easy or quick way to test deploying using these tools (mostly due to lack of experience with the tooling).

**Comment by wildum** [comment]
Same :/ I think that this is ok and we can rely on the community if something is wrong. @mattdurham what do you think?

**Review by mattdurham — APPROVED** [review]
Feels reasonable to me

**Pull request merged**